src/components/elements/dropdown.js

```diff
@@ -9,7 +9,7 @@ import { Flex } from 'components/containers'
 
 const Symbol = styled(Flex)`
     width: fit-content;
-    margin-top: 8px;
+    margin-top: px;
```

Comment:
Is this supposed to be `0`?

Reply:
No, I forgot to put it back to its original value after checking some stuff. Thanks